packages/explorer/src/app/api/world/route.ts

```diff
@@ -1,17 +1,16 @@
 import { AbiFunction, Address, Hex, createWalletClient, http, parseAbi } from "viem";
 import { getBlockNumber, getLogs } from "viem/actions";
-import { getRpcUrl } from "@latticexyz/common/foundry";
 import { helloStoreEvent } from "@latticexyz/store";
 import { helloWorldEvent } from "@latticexyz/world";
 import { getWorldAbi } from "@latticexyz/world/internal";
+import { chain } from "../../../common";
 
 export const dynamic = "force-dynamic";
 
 async function getClient() {
-  const profile = process.env.FOUNDRY_PROFILE;
-  const rpc = await getRpcUrl(profile);
   const client = createWalletClient({
-    transport: http(rpc),
+    chain,
+    transport: http(),
   });
```

Comment:
not blocking, but I wonder if we could pull this from the wagmi config using `@wagmi/core` to keep this all in one place?

```ts
import { getClient } from "@wagmi/core";
import { wagmiConfig, chainId } from "../common";

const client = getClient(wagmiConfig, { chainId });
```

Reply:
will work on that in a different PR as discussed